SECURITY-INSIGHTS.yml

```diff
@@ -20,6 +20,8 @@ security-testing:
   tool-name: Dependabot
   comment: |
     Dependabot is enabled for this repo.
+distribution-points:
+  - https://github.com/devfile/alizer
```

Comment:
In the docs for `distribution-points` it states:  "section contains a list of PURLs (Package URLs) for official project releases and distributions"

Do you think we should link to alizer releases instead? https://github.com/devfile/alizer/releases

Reply:
Yeah I thought about that. I was wondering if we can think of those releases as official project releases. I've made a quick check to see other examples and I found that many cases use the repo's URL in case of non-official project releases.

Reply:
Okay, if there are other repos following that same style then I think it is okay. If we need to change it we always can do that.

Reply:
Some examples I've first check:
- [envoyproxy/envoy/SECURITY-INSIGHTS.yml](https://github.com/envoyproxy/envoy/blob/a2673f749bb7a3f68f357e006e49b2143cc98b8f/SECURITY-INSIGHTS.yml#L43)
- [cilium/cilium/SECURITY-INSIGHTS.yml](https://github.com/cilium/cilium/blob/5863f8eeac279de8aa2e7934a6c3bd4ac57213ee/SECURITY-INSIGHTS.yml#L24)

I see there are other examples that are using the releases URL. Happy to updated if you think is better (@Jdubrick )

Reply:
@thepetk it looks like it takes an array input so we could add both the link to main repo and the releases section to cover all bases?

Reply:
Good idea but as I've already merged I'd say for now is ok to keep only the https://github.com/devfile/alizer one as it also includes the releases. WDYT?

Reply:
Good for me :)